features/admin.users.v1/components/user-profile.tsx

```diff
@@ -230,6 +231,12 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
         featureConfig?.users?.scopes?.update
     );
 
+    const entitlementConfig: FeatureAccessConfigInterface = useSelector(
+        (state: AppState) => state?.config?.ui?.features?.entitlement);
+    const hasRoleV3UpdateScopes: boolean = useRequiredScopes(entitlementConfig?.scopes?.update);
+    const updateUserRoleAssignment: (roleId: string, data: PatchRoleDataInterface) => Promise<AxiosResponse> =
```

Comment:
shall we rename as follows for consistency?

```suggestion
    const updateUserRoleAssignmentFunction: (roleId: string, data: PatchRoleDataInterface) => Promise<AxiosResponse> =
```